src/Data/Either.purs

```diff
@@ -282,3 +282,13 @@ note' f = maybe' (Left <<< f) Right
 -- | ```
 hush :: forall a b. Either a b -> Maybe b
 hush = either (const Nothing) Just
+
+-- | Turns an `Either` into a `Maybe`, by throwing eventual `Right` values away and converting
+-- | them into `Nothing`. `Left` values get turned into `Just`s.
+-- |
+-- | ```purescript
+-- | blush (Left "ParseError") = Just "Parse Error"
+-- | blush (Right 42) = Nothing
+-- | ```
+blush :: forall a b. Either a b -> Maybe b
```

Comment:
```diff
blush :: forall a b. Either a b -> Maybe b
blush :: forall a b. Either a b -> Maybe a
```

Reply:
Embarrassing

Reply:
I also noticed the "eventual" sounds like a Germanism and is probably meant to mean "possible". Can you confirm that?

Reply:
Yeah I think so, "eventual" has a bit of a different implication than German "eventuell" as I understand it - in English it implies that it will end up that way after a series of events. To me at least :smile:.